Public/Src/Engine/Scheduler/Artifacts/FileContentManager.cs

```diff
@@ -334,6 +334,18 @@ public void ReportInputContent(FileArtifact artifact, in FileMaterializationInfo
             }
         }
 
+        /// <summary>
+        /// Report the change affected outputs the pip. Only report those produced by the pip.
+        /// </summary>
+        public void ReportSourceChangeAffectedOutputs(AbsolutePath output, PipOutputOrigin origin, bool isFile)

```

Comment:
>ReportSourceChangeAffectedOutputs [](start = 20, length = 33)

Can you please consult Lance about this change in the `FileContentManager`?

Reply:
I removed this from FileContentManager.